nixos/modules/services/x11/picom.nix

```diff
@@ -301,13 +303,13 @@ in {
       };
 
       serviceConfig = {
-        ExecStart = "${pkgs.picom}/bin/picom --config ${configFile}";
+        ExecStart = "${getExe cfg.package} --config ${configFile}";
```

Comment:
It seems this https://github.com/NixOS/nixpkgs/blob/100a1550b0e7a64b960c625b656f9229bdef5f87/pkgs/applications/window-managers/picom/picom-next.nix#L32 overrides the meta.mainProgram because it's not recursively updating?

Reply:
>  I can do that as part of this PR or separately.

It's unrelated to the NixOS module, please do it in another PR.

Reply:
> This will need meta.mainProgram to be set in picom-next in order for the option to be useful. I can do that as part of this PR or separately.

FYI, I have made https://github.com/NixOS/nixpkgs/pull/249658 to fix the way picom-next adds meta.maintainers.